pkg/constants/env.go

```diff
@@ -33,7 +33,9 @@ const (
 
 	EnvPodName  = "OTEL_RESOURCE_ATTRIBUTES_POD_NAME"
 	EnvPodUID   = "OTEL_RESOURCE_ATTRIBUTES_POD_UID"
+	EnvPodIP    = "OTEL_POD_IP"
```

Comment:
is there an attribute name in the spec for the POD and NODE IPs? 

we should consider putting these under `OTEL_RESOURCE_ATTRIBUTES_`

Reply:
I opted not to use `OTEL_RESOURCE_ATTRIBUTES_` because I wasn't adding these values to the resource attributes. They are being added so that in Instrumentation users can to `http://$(OTEL_NODE_IP):4318` or `http://$(OTEL_POD_IP):4318` for their endpoints without having to specify any extra env vars.

Reply:
For semantic convention, there isn't a `k8s.pod.ip` or `k8s.node.ip` convention yet.